doc/moveit_cpp/launch/moveit_cpp_tutorial.launch.py

```diff
@@ -0,0 +1,101 @@
+    robot_description_kinematics = { 'robot_description_kinematics' : kinematics_yaml }
+
+    controllers_yaml = load_yaml('moveit2_tutorials', 'config/controllers.yaml')
+    moveit_controllers = { 'moveit_simple_controller_manager' : controllers_yaml,
+                           'moveit_controller_manager': 'moveit_simple_controller_manager/MoveItSimpleControllerManager'}
+
+    ompl_planning_pipeline_config = { 'ompl' : {
+        'planning_plugin' : 'ompl_interface/OMPLPlanner',
+        'request_adapters' : """default_planner_request_adapters/AddTimeOptimalParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints""" ,
+        'start_state_max_bounds_error' : 0.1 } }
+    ompl_planning_yaml = load_yaml('moveit_resources_panda_moveit_config', 'config/ompl_planning.yaml')
+    ompl_planning_pipeline_config['ompl'].update(ompl_planning_yaml)
+
+    # MoveItCpp demo executable
+    moveit_cpp_node = Node(name='moveit_cpp_tutorial',
+                               package='moveit2_tutorials',
+                               prefix='xterm -e',
+                               executable='moveit_cpp_tutorial',
+                               output='screen',
+                               parameters=[moveit_cpp_yaml_file_name,
+                                           robot_description,
+                                           robot_description_semantic,
+                                           kinematics_yaml,
+                                           ompl_planning_pipeline_config,
+                                           moveit_controllers])
+
+    # RViz
+    rviz_config_file = get_package_share_directory('moveit2_tutorials') + "/launch/moveit_cpp_tutorial.rviz"
+    rviz_node = Node(package='rviz2',
+                     executable='rviz2',
+                     name='rviz2',
+                     output='log',
+                     arguments=['-d', rviz_config_file],
+                     parameters=[robot_description,
+                                 robot_description_semantic])
+
+    # Static TF
+    static_tf = Node(package='tf2_ros',
+                     executable='static_transform_publisher',
+                     name='static_transform_publisher',
+                     output='log',
+                     arguments=['0.0', '0.0', '0.0', '0.0', '0.0', '0.0', 'world', 'panda_link0'])
+
+    # Publish TF
+    robot_state_publisher = Node(package='robot_state_publisher',
+                                 executable='robot_state_publisher',
+                                 name='robot_state_publisher',
+                                 output='both',
+                                 parameters=[robot_description])
+
+    # Fake joint driver
+    fake_joint_driver_node = Node(package='fake_joint_driver',
```

Comment:
We no longer depend on `fake_joint_driver` [see](https://github.com/ros-planning/moveit2/blob/main/moveit_demo_nodes/run_moveit_cpp/launch/run_moveit_cpp.launch.py#L135)